packages/site/src/routes/[dictionaryId]/contributors/+page.svelte

```diff
@@ -94,10 +95,24 @@
     startWith={helperType}
     let:data={contributors}>
     {#each contributors as contributor}
-      <div class="py-3">
+      <div class="py-3 flex flex-wrap items-center">
         <div class="text-sm leading-5 font-medium text-gray-900">
           {contributor.name}
         </div>
+        {#if $isManager}
+          <div class="w-1" />
+          <Button
+            onclick={() => {
+              if (confirm($t('misc.delete', { default: 'Delete' }) + '?')) {
+                removeDictionaryContributor(contributor, $dictionary.id);
+              }
```

Comment:
@Danble always confirm before a delete - it's way too easy to accidentally bump a button on mobile.